pkgs/development/python-modules/setuptools-rust/default.nix

```diff
@@ -11,12 +11,12 @@
 
 buildPythonPackage rec {
   pname = "setuptools-rust";
-  version = "0.11.6";
+  version = "0.12.1";
   disabled = isPy27;
```

Comment:
> Bump minimum Python version to Python 3.6.

Not sure if we should set it to disabled for python < 3.6.

Reply:
Although we don't have Python >3 && < 3.6 anymore, I don't think it hurts. Who knows, maybe someone has an old fork of nixpkgs and is overlaying newer packages ;). I'll update the PR.

Reply:
Fixed. Also fixed the passthru test.